### ISSUE TITLE
Weather API Shut Down

### ISSUE THREAD

**Issue by IanMitchell** [IanMitchell/aquarius#266]
```
  Weather Weather request for Katowice +49s
  Weather FetchError: request to https://query.yahooapis.com/v1/public/yql?q=select%20*%20from%20weather.forecast%20where%20u=%27f%27%20AND%20woeid%20in%20(select%20woeid%20from%20geo.places(1)%20where%20text=%22Katowice%22)&format=json failed, reason: getaddrinfo ENOTFOUND query.yahooapis.com query.yahooapis.com:443
```

**Comment by IanMitchell** [comment]
`Important EOL Notice: As of Thursday, Jan. 3, 2019, the weather.yahooapis.com and query.yahooapis.com for Yahoo Weather API will be retired. ` well damn.

**Comment by IanMitchell** [comment]
https://openweathermap.org/price as an alternative I guess?

**Comment by zaneneuschuler** [comment]
https://darksky.net/dev/docs Could possibly use this as well

**Comment by IanMitchell** [comment]
This is fixed in #275 albeit in a way that will require additional work at some point
